.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Release
+
+on:
+  pull_request:
+    types: [closed]
+    paths:
+      - '.github/project.yml'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: release
+    if: ${{github.event.pull_request.merged == true}}
+
+    steps:
+      - uses: radcortez/project-metadata-action@master
+        name: Retrieve project metadata
+        id: metadata
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          metadata-file-path: '.github/project.yml'
+
+      - uses: actions/checkout@v2
+
+      - name: Import GPG key
+        id: import_gpg
+        uses: crazy-max/ghaction-import-gpg@v3
+        with:
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.GPG_PASSPHRASE }}
+
+      - name: Install JDK 11
+        uses: joschi/setup-jdk@e87a7cec853d2dd7066adf837fe12bf0f3d45e52
+        with:
+          distribution: 'temurin'
+          java-version: 11
+          check-latest: true
+
+      - name: Configure Git author
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+      - name: Maven release ${{steps.metadata.outputs.current-version}}
+        run: |
+          mvn clean install -DskipTests
+          gpg --quiet --batch --yes --decrypt --passphrase="${{secrets.GPG_PASSPHRASE}}" --output maven-settings.xml .github/release/maven-settings.xml.gpg
+          git checkout -b release
+          mvn -X -Prelease -B release:clean release:prepare -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}} -s maven-settings.xml
+          git checkout ${{github.base_ref}}
+          git rebase release
+          mvn -X -Prelease -B release:perform -DskipTests -s maven-settings.xml
+
+          # update examples and tests
+          mvn versions:set -DnewVersion=${{steps.metadata.outputs.next-version}} -Pwith-examples,with-tests -DgenerateBackupPoms=false -DoldVersion=* -DgroupId=* -DartifactId=*
```

Comment:
We might want to call this: `https://github.com/dekorateio/dekorate/blob/main/scripts/ChangeVersion.java` so that we also update the readme.md.

Reply:
I kept the maven command to set the versions for the pom files (the benefit is that it enforces some rules).
And I've added the ChangeVersion.java command to update the README.md.